apis/core/hydra/projects.ttl

```diff
@@ -27,4 +31,6 @@ cc:ProjectsCollection
           code:link
             <file:handlers/cube-projects#post> ;
         ] ;
+      auth:scopes
+        scopes:pipelines-write
```

Comment:
Does the creation of a project require the ability to write to the pipelines or could this be seperated?

Reply:
Certainly. The entire permission is up for discussion

Reply:
I was not sure, if you had already some discussions.